mlir/test/Dialect/Linalg/vectorization-unsupported.mlir

```diff
@@ -112,6 +112,25 @@ module attributes {transform.with_named_sequence} {
 
 // -----
 
+func.func @conv2d(%3: tensor<1x64x58x58xf32>, %4:  tensor<64x64x3x3xf32>) {
```

Comment:
1. Could you add more examples? IIUC, none of the 2d and 3d convs would vectorize with this.
2. [nit] Big input tensors are IMHO a distraction. Why not use smaller sizes like in other examples in this file?

Reply:
> none of the 2d and 3d convs would vectorize with this.

I can add an additional conv3d test

> [nit] Big input tensors are IMHO a distraction

Agreed, I copy-pasted the actual crashing op without looking thoroughly in the context of this test. Will align the sizes like other tests in this file.